command/base_flags.go

```diff
@@ -10,6 +10,7 @@ import (
 	"strings"
 	"time"
 
+	"github.com/hashicorp/vault/sdk/helper/consts"
```

Comment:
For 1.9 we should be using math.MaxInt, it's built with Go 1.17

Reply:
Fixed it now